### ISSUE TITLE
[Bug]: Delete Blocks admin page shows extra checkboxes

### ISSUE THREAD

**Issue by diox** [mozilla/addons#15146]
### What happened?

- Add a block for a guid in the admin
- Try to delete it
- The delete block page shows a checkbox for `update_url_value` without a label, and a checkbox for `canned_reasons`. There is also a `None` that appears at the end.

### What did you expect to happen?

Not sure if the canned reasons is expected to be shown, but the one for the update url value certainly doesn't make sense to me, especially without the label. The `None` is likely due to the way we try to hide the `delayed_until` option, which doesn't make sense for delete block action.

See screenshot
![Screenshot 2024-11-06 at 14-37-57 Delete Block Django site admin](https://github.com/user-attachments/assets/d9216a10-4b58-4dd1-88e6-b90e0646b3f8)

### Is there an existing issue for this?

- [X] I have searched the existing issues

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-1203)

**Comment by ioanarusiczki** [comment]
Yes, that's something older. I remember filing this when last time changes were done to blocklisting.

**Comment by diox** [comment]
Oh yes, absolutely, I meant to add that this is not a new thing at all, I noticed this before we started tweaking the blocklist admin. I was hoping it would fix itself as we refactored the code but it doesn't look like it will, so I've filed it. 

I wasn't able to find an issue about it, but maybe it was just a comment somewhere. It's certainly not a very important issue, it's just confusing.

**Comment by ioanarusiczki** [comment]
np, couldn't find it either, maybe it was a comment after all, thanks for filing it 👍
